.github/workflows/ci_maintenance.yml

```diff
@@ -33,9 +33,7 @@ jobs:
           Shadow's dependencies may also need to be updated.
 
           \`\`\`bash
-          # "cargo upgrade" won't update the lock file for all
-          # dependencies, so "cargo update" is also needed
-          cd src && cargo upgrade --incompatible && cargo update
+          cd src && cargo update
           \`\`\`
```

Comment:
Why is the `cargo upgrade` removed?

Reply:
I got this:

```
$ cargo --version
cargo 1.75.0 (1d8b05cdd 2023-11-20)
$ cargo upgrade
error: no such command: `upgrade`

        Did you mean `update`?

        View all installed commands with `cargo --list`
        Find a package to install `upgrade` with `cargo search cargo-upgrade`
```

So I assumed `upgrade` was removed as a subcommand. But maybe my assumption is incorrect? What am I missing?

Reply:
Ah sorry, I should have included that in the documentation. The `cargo upgrade` command comes from [cargo-edit](https://crates.io/crates/cargo-edit). The maintainer hopes to get some version of this merged into cargo someday.

Running `cargo update` will not update things that may have breaking changes. For example if your Cargo.toml has `foo = 1.2.0` and foo has new versions 1.2.1 and 1.3.0, `cargo update` will only upgrade the dependency to 1.2.1. If you actually want the latest versions, you need to use `cargo upgrade` to upgrade to 1.3.0.

Reply:
OK, thank you! I've installed `cargo-edit` and will push an updated comment in `ci_maintenance.yml` to state the required package :)